pkg/network/infraconfigurators/masquerade.go

```diff
@@ -113,29 +107,6 @@ func (b *MasqueradePodNetworkConfigurator) discoverIPv6GatewayAndVmIp() error {
 	return nil
 }
 
-func (b *MasqueradePodNetworkConfigurator) GenerateDHCPConfig() *cache.DHCPConfig {
-	dhcpConfig := &cache.DHCPConfig{
-		Name: b.podNicLink.Attrs().Name,
-		IP:   b.vmIPv4Addr,
-		IPv6: b.vmIPv6Addr,
-	}
-	if b.vmMac != nil {
-		dhcpConfig.MAC = *b.vmMac
-	}
-	if b.podNicLink != nil {
-		dhcpConfig.Mtu = uint16(b.podNicLink.Attrs().MTU)
-	}
-	if b.vmGatewayAddr != nil {
-		dhcpConfig.AdvertisingIPAddr = b.vmGatewayAddr.IP.To4()
-		dhcpConfig.Gateway = b.vmGatewayAddr.IP.To4()
-	}
-	if b.vmGatewayIpv6Addr != nil {
-		dhcpConfig.AdvertisingIPv6Addr = b.vmGatewayIpv6Addr.IP.To16()
-	}
-
-	return dhcpConfig
-}
-
```

Comment:
I don't understand what removing this has to do with "Calculate domain on phase2" - which is the commit's msg.

Why are you deleting this ? I got the feeling I'm missing something, since I would expect compilation to fail once you stop implemented a method of an interface.